src/routes/api/assets/index.js

```diff
@@ -20,7 +20,7 @@ router.post(`/`, assetGenerationLimiter, require("./post"));
 
 router.get(`/`, require("./get"));
 
-router.get(`/:assetId`, require("./get"));
+router.get(`/:assetId`, require("./getById"));
```

Comment:
suggestion: We could undo this to avoid changing the same line of code in two different pull requests.

```suggestion
router.get(`/:assetId`, require("./get"));
```